e2e/docker_compose_confluent.yaml

```diff
@@ -46,7 +46,7 @@ services:
       CONNECT_KEY_CONVERTER: org.apache.kafka.connect.storage.StringConverter
       CONNECT_VALUE_CONVERTER: org.apache.kafka.connect.json.JsonConverter
       # CLASSPATH required due to CC-2422
-      CLASSPATH: /usr/share/java/monitoring-interceptors/monitoring-interceptors-7.4.1.jar
+      CLASSPATH: /usr/share/java/monitoring-interceptors/monitoring-interceptors-7.5.0.jar
```

Comment:
nit: since this is now also applied to `7.4.1` it could cause some trouble down the road (though all checks are passing and it seems `7.4.1` ignores this `CLASSPATH` since it doesn't exist, but it's still added to the config as per the logs). I suggest we only test with latest Confluent version, i.e., `7.5.0`, in these E2Es.

Reply:
D'oh.